pmoves/services/hi-rag-gateway-v2/hrm/sidecar.py

```
@@ -0,0 +1,350 @@
+        if self._loaded and self._model is not None and torch is not None:
+            try:
+                q_tensor = self._to_tensor(query_embedding)
+                if q_tensor is not None:
+                    if q_tensor.dim() == 1:
+                        q_tensor = q_tensor.unsqueeze(0).unsqueeze(0)
+                    elif q_tensor.dim() == 2:
+                        q_tensor = q_tensor.unsqueeze(0)
+                    with torch.no_grad():
+                        refined = self._model(q_tensor)
+                    result_tensor = refined.squeeze(0).detach().cpu().tolist()
+            except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)